_posts/2020-12-02-welcome-new-hire-getting-started-in-frontend-at-rimdev-updated.md

```diff
@@ -0,0 +1,92 @@
+image: https://images.unsplash.com/photo-1524530514346-eab43224489b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2800&q=80
+image_url: https://unsplash.com/photos/AqrD5QQ0ajM
+image_credit: "@ilumire"
+---
+
+<div class="ui info message">
+This is an updated post of the original <a href="/welcome-new-hire-getting-started-in-frontend-at-rimdev/">here</a>.
+</div>
+
+Whether you're new to the frontend at RIMdev or looking for more about our Frontend team, here's a quick overview of our team, tools, and [culture](https://github.com/ritterim/culture)!
+
+As a whole, RIMdev has always strived to use the technology that best suits the task. This post will give you a glimpse into our environment and tooling. Keep in mind, software changes and so do we.
+
+The majority of our frontend team are Mac users, so this post will seem Mac-biased, but if Windows is where you live, that's just fine! We have several Linux enthusiasts across the team as well, however, we don't support it for business use (that includes Linux subsystem for Windows).
+
+
+## Communicating
+
+Communication is at our core; we can accomplish more as our sum that we can as our parts, so we talk. About projects, goals, vision, and the path to get there. Slack is where we live, comment, post [gifs](https://giphy.com), [cat videos](https://www.youtube.com/watch?v=5dsGWM5XGdg), ya know.
+
+- [Slack](https://slack.com/)! We live here
+- [MS Teams](https://products.office.com/en-us/microsoft-teams/group-chat-software). Chat with non-slackers.
+- Outlook
+
+Looking to get up to speed on either? Here are some quick keyboard references for [Slack](https://slack.com/help/articles/201374536-Slack-keyboard-shortcuts) and [Teams](https://support.microsoft.com/en-us/office/keyboard-shortcuts-for-microsoft-teams-2e8e2a70-e8d8-4a19-949b-4c36dd5292d2.)
+
+## Environment
+
+RIMdev is alway on a journey to deliver a great, and constantly improving, frontend experience for our users. We leverage [VueJS](https://vuejs.org/), [webpack](https://webpack.js.org/) (although [ParcelJS](https://parceljs.org/) is gaining ground🤞), and [Sass](https://sass-lang.com/) heavily to build our Ritter Platform and family of static marketing sites. We've also built our own CSS framework [Platform UI](https://style.rimdev.io/) which continues to grow and be applied across our ecosystem. If you want to take a closer look, head over to [npm](https://www.npmjs.com/package/@ritterim/platform-ui) and grab a copy.
+
+- [Node](https://nodejs.org/en/download) & [npm](https://www.npmjs.com/)
+- [Homebrew](https://docs.brew.sh/Installation)🍺: Mac's missing package manager.
+- [Git](https://www.git-scm.com/) & [GitHub](https://github.com)
+- [RBenv](https://github.com/rbenv/rbenv): Ruby version management (for [Jekyll](https://jekyllrb.com/))
+- [Jekyll](https://jekyllrb.com/): Ruby-based static site generator.
+- [Hugo](https://gohugo.io/getting-started/quick-start/): Go lang based static site generator.
```

Comment:
```suggestion
- [Hugo](https://gohugo.io/getting-started/quick-start/): Golang based static site generator.
```

Reply:
>  The language is often referred to as Golang because of its domain name, golang.org , but the proper name is Go.

[wikipedia](https://en.wikipedia.org/wiki/Go_(programming_language)#:~:text=Go%20is%20syntactically%20similar%20to,the%20proper%20name%20is%20Go.) 
Changed to `[Go](https://golang.org/) based`